docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc

```diff
@@ -49,9 +49,9 @@ Once the zip bundle has been extracted, the directory structure should look simi
 
 == Windows
 
-To install on Windows 7 or higher use the ZIP bundle. Extract the CleanSpeak Management Interface ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as `CLEANSPEAK_HOME`. We suggest extracting this file to a directory such as `C:\inversoft` on Windows.
+To install on Windows 7 or higher, use the ZIP bundle. Extract the CleanSpeak Management Interface ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as CLEANSPEAK_HOME. We suggest extracting this file to a directory such as C:\inversoft on Windows.
```

Comment:
While we are in this path, let's update this to reflect our current min. requirements. I don't think Windows 7 is even supported by Microsoft any longer.